lib/galaxy/config/__init__.py

```diff
@@ -620,8 +620,36 @@ class GalaxyAppConfiguration(BaseAppConfiguration, CommonConfigurationMixin):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self._override_tempdir(kwargs)
+        self._configure_sqlalchemy20_warnings(kwargs)
         self._process_config(kwargs)
 
+    def _configure_sqlalchemy20_warnings(self, kwargs):
+        """
+        This method should be deleted after migration to SQLAlchemy 2.0 is complete.
+        To enable warnings, set `GALAXY_CONFIG_SQLALCHEMY_WARN_20=1`,
```

Comment:
Can you add this to API/selenium/integration tests ?

Reply:
and unit tests too!